### PR TITLE
feat: ZAPP-011/REGISTER_SCREEN

### PR DESCRIPTION
**Pull request by hajimemaste**

### CODE REVIEW

**Comment by hajimemaste** [comment]
a

File: lib/configs/routes_config.dart [hunk 1]
```diff
@@ -1,8 +1,10 @@
 import 'package:messenger/screens/home_screen.dart';
 import 'package:messenger/screens/login_screen.dart';
+import 'package:messenger/screens/register_screen.dart';
 import 'package:messenger/screens/splash_screen.dart';
 
 var routersConfig = {
   '//': (context) => const SplashScreen(),
   '/login': (context) => const LoginScreen(),
+  '/register': (context) => const RegisterScreen()
```
Comment (luan-thnh) [inline lib/configs/routes_config.dart]: Oce

File: android/app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -25,10 +25,29 @@
...
+                    "keyboard|keyboardHidden|screenLayout|screenSize|orientation"
+            android:label="@string/app_name" />
+        <activity
+            android:name="com.facebook.CustomTabActivity"
+            android:exported="true">
+            <intent-filter>
+                <action android:name="android.intent.action.VIEW" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+                <data android:scheme="@string/fb_login_protocol_scheme" />
```
Comment (luan-thnh) [inline android/app/src/main/AndroidManifest.xml]: oce

**Review by luan-thnh — DISMISSED** [review]

File: lib/screens/onboarding_screen.dart [hunk 3]
```diff
@@ -121,7 +122,13 @@ class _OnboardingScreenState extends State<OnboardingScreen> {
                   ],
                 ),
```
Comment (luan-thnh) [inline lib/screens/onboarding_screen.dart:123]: oce

**Review by luan-thnh — APPROVED** [review]

**Pull request merged**